.gitignore

```diff
@@ -32,4 +32,5 @@ CarthageUploads
 builds/
 Pods/
 IntegrationTests/CarthageIntegration/Cartfile.resolved
-scan_derived_data/
\ No newline at end of file
+scan_derived_data/
+generated_docs/
```

Comment:
we didn't use to have the docs folder generated, but it gets annoying when you run jazzy locally and end up with an uncommitted but not ignored folder lying around